Shopipy.ApiService/Program.cs

```diff
@@ -48,12 +49,13 @@
 }
 
 builder.Services.AddAutoMapper(typeof(UserMappingProfile), typeof(BusinessMappingProfile), typeof(ServiceMappingProfile), 
-    typeof(AppointmentMappingProfile), typeof(CategoryMappingProfile));
+    typeof(AppointmentMappingProfile), typeof(CategoryMappingProfile), typeof(ProductMappingProfile), typeof(ProductVariationMappingProfile));
```

Comment:
You can keep everything in one mapping profile. Also, I think, you only need to specify 1 mapping profile from every project here as automapper automatically scans for profiles.